### PR TITLE
Jormungandr: fix test_sp_to_sp for experimental

### PR DESCRIPTION
**Pull request by xlqian**

### CODE REVIEW

File: source/jormungandr/tests/journey_common_tests.py [hunk 1]
```diff
@@ -376,11 +376,22 @@ def test_sp_to_sp(self):
...
+            assert 'non_pt_walking' in journeys[0]['tags']
 
         # with street network activated
         response = self.query_region(query + "&max_duration_to_pt=1")
-        eq_(len(response['journeys']), 1)
+        if current_scenario in ('default', 'new_default'):
+            eq_(len(response['journeys']), 1)
+        elif current_scenario == 'experimental':
+            eq_(len(response['journeys']), 2)
+
```
Comment (TeXitoi) [inline source/jormungandr/tests/journey_common_tests.py:394]: I'd prefer to check that the first (resp last) section is a public_transport section with from (resp to) = request from (resp to).

**Review by TeXitoi — CHANGES_REQUESTED** [review]

File: source/jormungandr/tests/journey_common_tests.py (hunk 1 above)
Comment (TeXitoi) [reply source/jormungandr/tests/journey_common_tests.py:394]: Sorry, that's not the problem. But we shouldn't have this journey if we want to be isofunctional. Maybe that should be done in another ticket.

**Review by TeXitoi — APPROVED** [review]

File: source/jormungandr/tests/journey_common_tests.py (hunk 1 above)
Comment (xlqian) [reply source/jormungandr/tests/journey_common_tests.py:394]: i created a ticket for that

**Comment by TeXitoi** [comment]
Will be fixed globally in another PR

**Pull request closed without merging**